src/Components/ToggleSave/ToggleSave.css

```diff
@@ -1,15 +1,33 @@
 .toggle-save-container {
-    cursor: pointer;
-    display: inline-block;
-  }
-  
-  .toggle-image {
-    width: 40px;
-    height: auto;
-    transition: transform 0.3s;
-    
-  }
-  
-  .toggled-on {
-    transform: rotate(20deg);
-  }
\ No newline at end of file
+  position: relative;
+  cursor: pointer;
+  display: inline-block;
+  z-index: 2; 
```

Comment:
The power of z-indeX!

Reply:
Do we still need the z index, with the changes you implemented?